src/Toml/Bi/Combinators.hs

```diff
@@ -150,6 +179,15 @@ text = match _Text
 string :: Key -> TomlCodec String
 string = match _String
 
+codecShow :: (Show a, Read a, Typeable a) => Key -> TomlCodec a
+codecShow = match _Show
```

Comment:
I think that this codec should be named just `show`. This modules is supposed to be exported qualified so people will use `Toml.show` which is better than `Toml.codeShow`

Reply:
Also, I'm starting to think that maybe it's better to call it `read` instead of `show`? We use both, but I guess `read` is more explicit. What do you think?